projects/packages/videopress/src/client/block-editor/blocks/video/components/player/index.native.js

```diff
@@ -148,17 +148,28 @@ export default function Player( { isSelected, attributes } ) {
 
 	let overlay = ! isSelected && <View style={ style[ 'videopress-player__overlay' ] } />;
 
-	// On Android render an overlay to send the embed markup to a native WebView.
-	if ( isSelected && IS_ANDROID ) {
-		overlay = (
-			<View style={ style[ 'videopress-player__overlay' ] }>
-				<Pressable
-					style={ style[ 'videopress-player__open-embed-button' ] }
-					onPress={ () => {
-						requestEmbedFullscreenPreview( html, title );
-					} }
-				/>
-			</View>
+	let sandbox = (
+		<SandBox
+			html={ html }
+			onWindowEvents={ { message: onSandboxMessage } }
+			viewportProps="user-scalable=0"
+			testID="videopress-player"
+			onLoadEnd={ onLoadEnd }
+		/>
+	);
+
+	// On Android, we need to open the embed preview in a Native WebView
+	if ( IS_ANDROID ) {
+		sandbox = (
+			<Pressable
+				style={ style[ 'videopress-player__open-embed-button' ] }
+				onPress={ () => {
+					requestEmbedFullscreenPreview( html, title );
+				} }
+			>
+				<View style={ style[ 'videopress-player__overlay' ] } />
```

Comment:
I understand we add the overlay here to disable touch events in the WebView, is this accurate?

Reply:
I'm wondering if we could reuse the `overlay` constant defined above here, WDYT?

Reply:
> I understand we add the overlay here to disable touch events in the WebView, is this accurate?

Correct, I noticed that the video sometimes starts playing as the WebView is opening. Which is ironic since we need the web view to probably handle the touch events 😅

Reply:
fixed in 2d886e2aa3e954fc02ff9a00d35458abe9d424a6

Reply:
> I'm wondering if we could reuse the overlay constant defined above here, WDYT?

I ended up removing the `overlay` variable so it's just the single `View` now.